lib/vis_milk2/ns-eel2/Makefile

```diff
@@ -0,0 +1,120 @@
+CC=gcc
```

Comment:
please drop this, it's not used and will never be

Reply:
the Makefile isn't used at all, only included it as it makes diffing easier if there's ever any upstream work we're interested in.